docs/source/CONTRIBUTING.md

```diff
@@ -109,9 +110,9 @@ to get used to the basic usage.
 
 Now, setting up a development environment is super simple! Additional info if you run into trouble: [Poetry Environments]
 
-Step 1: [Install Poetry]  
-Step 2: Run ``poetry install``  
-Step 3: Optionally Run ``poetry shell``  
+Step 1: [Install Poetry]
+Step 2: Run ``poetry install``
+Step 3: Optionally Run ``poetry shell``
```

Comment:
```suggestion
Step 1: [Install Poetry]  
Step 2: Run ``poetry install``  
Step 3: Optionally Run ``poetry shell``  
```
It's important that we do not remove trailing whitespace in Markdown files. See this comment: https://github.com/godaddy/tartufo/pull/124#discussion_r512994249

Reply:
hrm, looks like intellij is not respecting the .editorconfig